tests/spec/examples.py

```diff
@@ -403,7 +403,7 @@ def it_displays_correctly_when_the_service_is_up_json(self, in_example_dir):
                 'state': 'ready',
             },
         }
-        assert stderr == b''
+        assert stderr == ''
```

Comment:
For a non test API that makes a lot of sense. For a test API though, that requires every test to decode these values, which adds some cruft.